src/types/modifications/CCUS.jl

```diff
@@ -3,12 +3,30 @@
 
     CCUS(;file, groupby, co2_scalar=1e5, co2_step_quantity_limit=1e9)
 
-This is a [`Modification`](@ref) that sets up markets for carbon captured by generators.  
+This is a [`Modification`](@ref) that sets up markets for carbon captured by generators.
+
+### Keyword Arguments    
 * `file` - a file to the table containing markets/prices for buying/selling carbon dioxide.  See the `ccus_paths` table below, or [`summarize_table(::Val{:ccus_paths})`](@ref)
 * `groupby` - a `String` indicating how markets are grouped.  I.e. "state".
 * `co2_scalar` - a `Float64` for how much to scale the co2 variables by.  This helps with numerical instability, given that some CO₂ steps can be very large and could bloat the RHS and bounds range of the model.  A good rule of thumb is that this should be no less than `1e4` times smaller than `co2_step_quantity_limit`.
 * `co2_step_quantity_limit` - A `Float64` for the maximum quantity of CO₂ that can be stored in any step.
 
+### Tables Added
+* `<name>_ccus_paths` - contains all the pathways to selling CO₂. The table includes the producing region, the sequstering region and the type of storage, market steps, the quantity available for storage, and the costs of storage and transport.
+
+### Table Columns Added
+* `(:ccus_paths, :path_idx)` - the index of this path. 
+* `(:ccus_paths, :price_total)` - the cost of transporting and storing a short ton of CO₂ in this storage pathway.
+* `(:ccus_paths, :stored_co2)` - the amount of CO₂ stored via this storage path, measured in short tons.
+* `(:ccus_paths, :storer_cost)` - total storage cost of CCUS via this pathway, from the perspective of the sequesterer.
+* `(:ccus_paths, :storer_revenue)` - total revenue earned from storage for CCUS via this pathway.  This is the amount paid by the EGU's to the sequesterer, equal to the clearing price minus the transport cost.
+* `(:ccus_paths, :storer_profit)` -  total profit earned by storing carbon via this pathway, equal to the revenue minus the cost.
+* `(:gen, :capt_co2)` - the rate of capture of CO2 (calculated from emis_co2 and capt_co2_percent).
```

Comment:
Minor thing here.  In the julia REPL, looks like markdown is mis-interpreting your underscores in emis_co2 and capt_co2_percent as you trying to underline.  For example `_this_` is interpreted as _this_.  To correct this, you can surround the column names that contain underscores with a tick mark (i.e. ` )

Reply:
Hi Ethan, thanks for the review! I made all the amendments and re-committed and seems like everything is in order. thanks!